massdriver.yaml

```diff
@@ -35,78 +35,31 @@ params:
             min_size: 1
             max_size: 10
   required:
-    - kubernetes_version
+    - cluster
     - node_groups
     - observability
   properties:
-    observability:
+    cluster:
```

Comment:
Note this is a breaking change in the DB.

Currently there are 3 packages of this bundle using the public version. Two are ours, and one is a customers (but is currently unconfigured).  You will need to release this immediately, or you will need to coordinate a DB migration w/ the platform team.

Reply:
I can have it tested/released by today